### DISCUSSION TITLE
Multi region and multi DC installation

### DISCUSSION THREAD

**Discussion by RagavMaddali** [deployKF/deployKF#207] [Questions & Answers]
To ensure HA of the service, we wanted to install deploykf in multiple region and multiple DC. Requesting your help in implementing HA for the services.

**Comment by thesuperzapper** [comment]
deployKF runs on a Kubernetes cluster, and multi-region support would need to be implemented at the cluster level, by your Kubernetes distribution, (like EKS, GKE, or similar).

_PS: One important thing to remember about multi-region Kubernetes clusters is that storage is always a problem. That is, during an outage of one region, even if the cluster itself remains up, if you have data on PVCs that are stored in one region, Pods wont be able to start on Nodes in the other region (this is most applicable for Kubeflow Notebooks, as with Pipelines, you can use an external object store)._
